sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/analysis/Analyzer.scala

```diff
@@ -886,7 +886,7 @@ class Analyzer(
             Some(relation)
           }
         case Some(table) =>
-          Some(DataSourceV2Relation.create(table))
+          Some(DataSourceV2Relation.create(table, Some(catalog), Seq(ident)))
```

Comment:
You need `newIdent` here?
Also `catalog` will be `CatalogManager.SESSION_CATALOG_NAME` here